packages/vscode-extension/src/handlers.ts

```diff
@@ -819,14 +819,13 @@ async function openMarkdownHandler() {
     const workspaceFolder = workspace.workspaceFolders[0];
     const workspacePath: string = workspaceFolder.uri.fsPath;
     let targetFolder: string | undefined;
-    showLocalDebugMessage();
     if (await isMigrateFromV1Project(workspacePath)) {
       targetFolder = workspacePath;
     } else if (await isSPFxProject(workspacePath)) {
-      showChangeLocationMessage();
```

Comment:
One thing to note is that the local debug notification will not pop up for migrate v1 project. I think  that's acceptable. Add @XiaofuHuang for awareness.